README.md

```diff
@@ -4,53 +4,29 @@
 
 > Description the project.
 
-
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- JavaScript
 
 ## Live Demo (if available)
 
 [Live Demo Link](https://livedemo.com)
 
```

Comment:
- [ ] Kindly add the link for a video presenting the features of the project.
- [ ] Kindly update the link to your deployed project in the README.